src/main/java/com/couchbase/lite/store/SQLiteStore.java

```diff
@@ -2315,7 +2339,7 @@ protected int pruneRevsToMaxDepth(int maxDepth) throws CouchbaseLiteException {
         try {
             beginTransaction();
             for (Long docNumericID : toPrune.keySet())
-                outPruned += pruneDocument(docNumericID, toPrune.get(docNumericID).intValue() + 1);
+                outPruned += pruneDocument("?", docNumericID, toPrune.get(docNumericID).intValue() + 1);
```

Comment:
In iOS implementation, pruneDocument()'s docID parameter is only used for logging.
https://github.com/couchbase/couchbase-lite-ios/blob/2d6e6355e46cacab46d404bfb5e7ceaa1a92a7c9/Source/CBL_SQLiteStorage.m#L2093

For Java, we could remove this parameter.